WISE-Community/WISE-Client
# Dialog guidance && (and) operator

## Changes
- Change FeedbackRule.rule array to FeedbackRule.expression string
- Add support for && operator in the FeedbackRule.expression

## Test
- change every "rule" array to "expression" string in feedbackRules block in the component content
```
change
"rule": ["idea1"]
to:
"rule": "idea1"
```
- add '&&' (and) expression FeedbackRules and see if they work
```
{
  "expression": "idea1 && idea2",  // expression can contain spaces
  "feedback": "you hit idea1 and idea2"
},
{
  "expression": "idea2&&idea3&&idea6", 
  "feedback": "you hit idea2 and idea3 and idea6"
}
```
- make sure that other expressions still work:
  - ```"expression": "ideaX"```
  - ```"expression": "isFinalSubmit"```
  - ```"expression": "isSecondToLastSubmit"```
  - ```"expression": "isDefault"```


Closes #345
